manpages/lumina-xconfig.8

```diff
@@ -1 +1,37 @@
-Dummy file
+.Dd November 3, 2017
+.Dt LUMINA-XCONFIG 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-xconfig
+.Nd a graphical front-end to xrandr.
+It is used to manage monitor configurations.
+
+.Sh SYNOPSIS
+.Nm
+.Op Fl -reset-monitors
+
+.Sh DESCRIPTION
+.Nm
+opens the graphical utility for managing monitor configurations.
+It also requires the runtime utility xrandr.
+The xrandr utility provides options such as rotate, primary screen,
+resolution, and more.
+.Nm
+is able to take one optional argument.
+.Bl -tag -width indent
+.It Ic --reset-monitors
+Resets the configuration to default.
+.El
+
```

Comment:
You may want to add an .Sh DEPENDENCIES section with xrandr listed, so the user can quickly see that it is needed.